pkg/util/timeUtil.go

```diff
@@ -0,0 +1,16 @@
+package util
+
+import (
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"time"
+)
+
+// ToTime returns a time.Time from a *timestamp.Timestamp.
+func ToUTCTime(t *timestamp.Timestamp) time.Time {
```

Comment:
Mentioned before but these two are in the `ptypes` package of protobuf

Reply:
Right, I found the function in ptypes and replaced with it. Resolved other comments as well.